pegasus-spark-bulkloader/src/main/java/com/xiaomi/infra/pegasus/spark/bulkloader/BulkLoadInfo.java

```diff
@@ -0,0 +1,16 @@
+package com.xiaomi.infra.pegasus.spark.bulkloader;
+
+public class BulkLoadInfo {
+
+  private String cluster;
+  private int app_id;
+  private String app_name;
+  private int partition_count;
```

Comment:
```suggestion
  private int appID;
  private String appName;
  private int partitionCount;
```

Reply:
The name which will be write to fds need be same with server define

Reply:
Ok. We can add a function `toJSON` that converts this class to JSON string.

```
String toJSONString() {
  return JSON.toJSONString(this);
}
```